src/sinol_make/commands/inwer/inwer_util.py

```diff
@@ -0,0 +1,91 @@
+import glob
+import os
+import sys
+
+import argparse
+
+from sinol_make import util
+from sinol_make.commands.inwer import TestResult, TableData
+from sinol_make.helpers import compile, package_util
+from sinol_make.helpers import compiler
+from sinol_make.interfaces.Errors import CompilationError
+
+
+def get_inwer(task_id: str, path = None) -> str or None:
```

Comment:
```suggestion
def get_inwer_path(task_id: str, path = None) -> str or None:
```
Some additional changes might be needed for this one.